roles/nginx/tasks/main.yml

```diff
@@ -33,9 +33,10 @@
     force: yes
 
 - name: Move h5bp configs
-  command: cp -R {{ nginx_path }}/h5bp-server-configs/h5bp {{ nginx_path }}/h5bp
-  args:
-    creates: "{{ nginx_path }}/h5bp/"
+  command: rsync -ac --info=NAME {{ nginx_path }}/h5bp-server-configs/h5bp/ {{ nginx_path }}/h5bp
```

Comment:
We need the `--delete` flag to remove deleted files from `{{ nginx_path }}/h5bp`

Without `--delete`:
```bash
$ tree src dest
src
├── BBB.txt
└── CCC.txt
dest
├── AAA.txt
└── BBB.txt

$ rsync -ac --info=NAME ./src/ ./dest/
./
CCC.txt

$ tree src dest
src
├── BBB.txt
└── CCC.txt
dest
├── AAA.txt
├── BBB.txt
└── CCC.txt
```
AAA.txt still there.

With `--delete`:
```bash
$ tree src dest
src
├── BBB.txt
└── CCC.txt
dest
├── AAA.txt
└── BBB.txt

$ rsync -ac --delete --info=NAME ./src/ ./dest/
deleting AAA.txt
deleting .DS_Store
./
CCC.txt

$ tree src dest
src
├── BBB.txt
└── CCC.txt
dest
├── BBB.txt
└── CCC.txt
```
AAA.txt is gone.

Reply:
Excellent! Thanks @TangRufus!